gcloud/bigquery/table.py

```diff
@@ -50,6 +83,32 @@ def path(self):
         return '%s/tables/%s' % (self._dataset.path, self.name)
 
     @property
+    def schema(self):
+        """Table's schema.
+
+        :rtype: list of ``SchemaField``
+        :returns: fields describing the schema
+        """
+        return list(self._schema)
+
+    @schema.setter
+    def schema(self, value):
+        """Update table's schema
+
+        :type value: list of ``SchemaField``
+        :param value: fields describing the schema
+
+        :raises: TypeError if 'value' is not a sequence, or ValueError if
+                 any item in the sequence is not a SchemaField
+        """
+        fields = list(value)
+        if len(fields) > 0:
+            types = set([type(field) for field in fields])
```

Comment:
No need to create a new list or a set:

```python
all(isinstance(field, SchemaField) for field in value)
```

Probably irrelevant given the tiny sizes of inputs.